portfolio/src/main/webapp/index.html

```diff
@@ -152,7 +151,8 @@ <h3>My Portfolio Site </h3>
     <div class="blankspace"></div> <!-- necessary to allow for scrolling to blog section" -->
 
     <footer>
-        <button onclick="msg()" style="float:right; margin-bottom:20px;"><i class="fa fa-commenting-o fa-4x"aria-hidden="true"></i></button>
+        <
```

Comment:
Unmatched angle bracket.

Reply:
yup I caught that in a newer branch, didn't update here